src/main.rs

```diff
@@ -52,12 +52,12 @@ fn run(cli: Cli) -> Result<i32> {
         command = command.with_cwd();
     }
 
-    let envs = cli.env.into_iter().flatten().collect::<Vec<_>>();
+    let envs = cli.env;
```

Comment:
You can do away with these assignments now, just read the field directly since we're not constructing a separate vec.

Reply:
Just felt right to keep it in a new variable, but I can change that if you like